tests/test_class.cpp

```diff
@@ -400,13 +400,13 @@ template <int N> class BreaksBase { public:
 template <int N> class BreaksTramp : public BreaksBase<N> {};
 // These should all compile just fine:
 typedef py::class_<BreaksBase<1>, std::unique_ptr<BreaksBase<1>>, BreaksTramp<1>> DoesntBreak1;
-typedef py::class_<BreaksBase<2>, BreaksTramp<2>, std::unique_ptr<BreaksBase<2>>> DoesntBreak2;
-typedef py::class_<BreaksBase<3>, std::unique_ptr<BreaksBase<3>>> DoesntBreak3;
-typedef py::class_<BreaksBase<4>, BreaksTramp<4>> DoesntBreak4;
-typedef py::class_<BreaksBase<5>> DoesntBreak5;
-typedef py::class_<BreaksBase<6>, std::shared_ptr<BreaksBase<6>>, BreaksTramp<6>> DoesntBreak6;
-typedef py::class_<BreaksBase<7>, BreaksTramp<7>, std::shared_ptr<BreaksBase<7>>> DoesntBreak7;
-typedef py::class_<BreaksBase<8>, std::shared_ptr<BreaksBase<8>>> DoesntBreak8;
+using DoesntBreak2 = py::class_<BreaksBase<2>, BreaksTramp<2>, std::unique_ptr<BreaksBase<2> > >;
+using DoesntBreak3 = py::class_<BreaksBase<3>, std::unique_ptr<BreaksBase<3> > >;
+using DoesntBreak4 = py::class_<BreaksBase<4>, BreaksTramp<4> >;
+using DoesntBreak5 = py::class_<BreaksBase<5> >;
+using DoesntBreak6 = py::class_<BreaksBase<6>, std::shared_ptr<BreaksBase<6> >, BreaksTramp<6> >;
+using DoesntBreak7 = py::class_<BreaksBase<7>, BreaksTramp<7>, std::shared_ptr<BreaksBase<7> > >;
+using DoesntBreak8 = py::class_<BreaksBase<8>, std::shared_ptr<BreaksBase<8> > >;
```

Comment:
@YannickJadoul This reads _much_ better now, doesn't it? Having the type name on the left, like everything else in C++, is nicer and more consistent (and in newer C++, there are some further tricks to using). Just ignoring typedef as if it never existed is better than mixing them randomly :)

Reply:
Yes, OK, there's something to be said for that, indeed :-) (It looks drastic to now change it everywhere, but it will be better.)

Now that you point it out, what about these changes of `>>>` -> `> > >`?

Reply:
I can manually fix those, I assume it is trying to be C++03 compatible. When modernizing C++11 code, hmm...

Reply:
Not sure if it's worth your effort (and the extra annotations), though...

Reply:
Oh, no, it would maybe not correct those back? Wait, let me do the effort, then ;-)

Reply:
OH. I'm already too late; this view was outdated :-| Thanks, @henryiii! :-)